packages/eslint-config/framework.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path')
+
+const findUp = require('findup-sync')
+
+const findBabelConfig = (cwd = process.cwd()) => {
+  const configPath = findUp('babel.config.js', { cwd })
+  if (!configPath) {
+    throw new Error(`Eslint-parser could not find a "babel.config.js" file`)
+  }
+  return configPath
+}
+
+// Just configure the babel options
+// The framework is configured with babel.config.js
+// While projects are configured programatically from rwjs/internal
+module.exports = {
+  extends: path.join(__dirname, 'shared.js'),
+  parserOptions: {
+    babelOptions: {
+      configFile: findBabelConfig(),
+    },
+  },
+}
```

Comment:
@dac09 could all this code live in `./.eslintrc.js` instead of a separate file here?

Reply:
Nope, because of the "findUp" dependency

Reply:
But will you need findUp since they're both in root?

Reply:
Hmmm I don't know, I'll try, but I wanted to minimise the changes here... I find lint configuration boring and impossible to debug.

Reply:
Understood. When I was tracing the changes and saw the files "shared" and "framework", I assumed two things: 1) there would be a "project" and 2) there would be eslint config in framework. When I saw that this file is specifically handling associated babel config for the framework, that's what got me thinking about whether or not it could be simplified further.

So all that mostly for discussion. Maybe what all this really needs is documentation in the package README.